core/domain/draft_upgrade_services.py

```diff
@@ -164,6 +164,23 @@ def extract_html_from_draft_change_list(draft_change_list):
 class DraftUpgradeUtil(python_utils.OBJECT):
     """Wrapper class that contains util functions to upgrade drafts."""
 
+    @classmethod
+    def _convert_states_v34_dict_to_v35_dict(cls, draft_change_list):
+        """Converts draft change list from state version 34 to 35. State
+        version 35 upgrades all explorations that use the MathExpressionInput
+        interaction to use one of AlgebraicExpressionInput,
+        NumericExpressionInput, or MathEquationInput interactions. There should
+        be no changes to the draft for this migration.
+
+        Args:
+            draft_change_list: list(ExplorationChange). The list of
+                ExplorationChange domain objects to upgrade.
+
+        Returns:
+            list(ExplorationChange). The converted draft_change_list.
+        """
+        return draft_change_list
```

Comment:
I thought the delete would be done here? Or is  that after the audit job is run?